.eslintrc.json

```diff
@@ -27,11 +24,41 @@
         "react/display-name": 0,
         "react/prop-types": 0,
         "react/react-in-jsx-scope": 0,
-        "semi": ["error", "never"]
+        "semi": ["error", "never"],
+        "no-debugger": "error",
+        "eol-last": ["error", "always"]
     },
     "settings": {
         "react": {
             "version": "detect"
         }
-    }
+    },
+    "overrides": [{
+        "files": ["src/**/*.tsx", "src/**/*.ts"],
+        "parser": "@typescript-eslint/parser",
+        "extends": "plugin:@typescript-eslint/recommended",
+        "plugins": ["@typescript-eslint"],
+        "parserOptions": {
+            "project": "tsconfig.json"
+        },
+
+        "rules": {
```

Comment:
Can we start with the defaults?

Reply:
You mean the rules specified some lines above? These simply overwrite the previous one. So anything added their will be used here or otherwise specified in this section.

Reply:
I tried it out on my branch without custom rules for typescript and got reasonable results.

The warnings that it generated seemed like good things to fix but were not show stoppers.

Reply:
Done